src/error-suppressions/error-suppressions.repository.ts

```diff
@@ -83,7 +83,7 @@ export class ErrorSuppressionsRepository extends Repository<EsSpec> {
     if (locations) {
       query.andWhere(
         `string_to_array(es.locations, ',') && string_to_array(:locations, ',')`,
-        { locations: locations.join(',') },
+        { locations: locations },
```

Comment:
The `string_to_array` function splits a string into an array. Since the `locations` request parameter is an array, the original code correctly used `join(',')` to stringify it. Alternatively, the array could be passed in directly, e.g.:
```typescript
      query.andWhere(
        `string_to_array(es.locations, ',') && :locations`,
        { locations },
      );
```